app/controllers/api/v2/accredited_provider_training_providers_controller.rb

```diff
@@ -0,0 +1,29 @@
+module API
+  module V2
+    class AccreditedProviderTrainingProvidersController < ApplicationController
+      before_action :build_recruitment_cycle
+      before_action :build_provider
+
+      def index
+        authorize @provider, :can_list_training_providers?
+        providers = @provider.training_providers
+
+        render jsonapi: providers, include: params[:include]
+      end
+
+    private
+
+      def build_recruitment_cycle
+        @recruitment_cycle = RecruitmentCycle.find_by(
+          year: params[:recruitment_cycle_year],
+        ) || RecruitmentCycle.current_recruitment_cycle
+      end
+
+      def build_provider
+        @provider = @recruitment_cycle.providers.find_by!(
```

Comment:
Correct me if I'm wrong, but I *think* we need to be using the policy scope here. See https://github.com/DFE-Digital/manage-courses-backend/blob/2731-test-setup-helper/app/controllers/api/v2/providers_controller.rb#L14 for an example.

Reply:
Nope. Having thought about this some more, this is ok because we have `authorize @provider, :can_list_training_providers?` in `index`.